src/engraving/rw/write/twrite.cpp

```diff
@@ -1551,6 +1551,7 @@ void TWrite::writeProperties(const Spanner* item, XmlWriter& xml, WriteContext&
     if (ctx.clipboardmode()) {
         xml.tagFraction("ticks_f", item->ticks());
     }
+    writeProperty(item, xml, Pid::PLAY);
```

Comment:
But now it would be written twice for some spanners, so you still need to remove the `writeProperty` calls for subclasses